controllers/teams.js

```diff
@@ -500,7 +506,7 @@ router.get('/:teamId', async (req, res, next) => {
 			events = [];
 		}
 
-		let test = course.user.permissions.includes('EDIT_ALL_FILES')
+		// const test = course.user.permissions.includes('EDIT_ALL_FILES');
```

Comment:
was ist hier los?

Reply:
Unbenutzte Variable. Ich weiß nicht, ob die noch Sinn macht, daher nur auskommentiert.

Reply:
Wenn unbenutzt würde ich sagen: raus damit! Kannst ja nochmal prüfen, ob es in der Datei noch anderen Code gibt, der auch auskommentiert ist und die Variable verwendet. Aufgrund der Benennung würde ich aber vermuten, dass jemand nach dem testen vergessen hat, diese wieder zu entfernen.